configure.ac

```diff
@@ -87,6 +87,13 @@ AC_ARG_ENABLE([kvm],
       [enable_kvm=yes])
 AM_CONDITIONAL([WITH_KVM], [test x"$enable_kvm" = xyes])
 
+AC_ARG_ENABLE([libkvmi],
```

Comment:
I don't think that libkvmi should be enabled or disabled.

To me it **replaces** the current KVM driver of Libvmi, so if the user configured Libvmi to have KVM support, just check that libkvmi is here, or throw an error.

And in the driver there is no need for `HAVE_LIBKVMI` in the same way.

I would like to have @tklengyel opinion too.

Reply:
I would be all for replacing the existing KVM driver with this, but there are features built on top of the existing KVM driver that would need to be removed in that case, like SHM. IMHO that feature was always experimental and not well maintained so removing it should be fine.